docker-compose.yml.tpl

```diff
@@ -12,6 +16,18 @@ panteras:
     FQDN:                    "${FQDN}"
     GOMAXPROCS:              "${GOMAXPROCS}"
 
+    SERVICE_8500_TAGS: haproxy
+    SERVICE_8500_NAME: consul
```

Comment:
yes, but I'm adding tag haproxy only for port 8500 - So you can access UI(8500) via browser without port, but